api/WindowOrWorkerGlobalScope.json

```diff
@@ -1067,10 +1061,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/WindowOrWorkerGlobalScope/setInterval",
           "support": {
             "chrome": {
-              "version_added": "1"
+              "version_added": "54"
```

Comment:
[This CL](https://chromium.googlesource.com/chromium/src/+/387fa9a4b0c7cbbcac73ab8bc201d5c36e366ab6) is as close as I can get to anything definitive. The time frame is Chrome 30 for desktop and Android. Webview should say  true.